datasets/br/br_tcu_disqualified/crawler.py

```diff
@@ -0,0 +1,73 @@
+from zavod import Context, helpers as h
+from rigour.ids.stdnum import CPF
```

Comment:
it's not working for me at the moment - it looks like it should maybe be rigour.ids.stdnum_ - the underscore is probably added to avoid conflict